specification/maps/data-plane/Microsoft.Maps/DEM/preview/1.0/elevation.json

```diff
@@ -133,8 +133,8 @@
   "paths": {
     "/elevation/point/{format}": {
       "get": {
-        "summary": "Get elevation data on one or more points.",
-        "description": "**Applies to**: S1 pricing tier.\nThe Get Data for Points API provides elevation data for one or more points.  A point is defined in lat,long coordinate format. Due to the URL character  length limit of 2048, it's not possible to pass more than 100 coordinates  as a pipeline delimited string in a URL GET request. If you intend to pass  more than 100 coordinates as a pipeline delimited string, use the [POST Data\n For Points](https://docs.microsoft.com/en-us/rest/api/maps/elevation/postdataforpoints).",
+        "summary": "**Get Elevation Data on One or More Points**\n",
```

Comment:
Please remove all formatting from the summaries (`**` and `\n`) as this breaks the consistent look-and-feel for all SDK documentation.

Reply:
Thanks Joel.  The doc is updated based on your comments.